core/eval.go

```diff
@@ -851,6 +851,14 @@ func executeCommand(cmd *RedisCmd, c *Client) []byte {
 		return evalQINTLEN(cmd.Args)
 	case "QINTPEEK":
 		return evalQINTPEEK(cmd.Args)
+	case "BFINIT":
+		return evalBFInit(cmd.Args)
+	case "BFADD":
+		return evalBFAdd(cmd.Args)
+	case "BFEXISTS":
+		return evalBFExists(cmd.Args)
+	case "BFINFO":
+		return evalBFInfo(cmd.Args)
```

Comment:
Fix the case. We are following Command in caps so, evalBFInit becomes `evalBFINIT`. Alter all the names.

Reply:
Done.